li-hadoop-plugin/src/main/groovy/com/linkedin/gradle/liazkaban/LiAzkabanDslCompiler.groovy

```diff
@@ -31,6 +31,7 @@ class LiAzkabanDslCompiler extends AzkabanDslCompiler {
 
   private static final String BANGBANG_TEMPLATE = "LiBangBangJob.template";
   private LiHadoopShellCommandFactory bangBangCommandFactory;
+  private static final String PIG_JAVA_OPTS = "env.PIG_JAVA_OPTS";
```

Comment:
Could you put this private static final string next to the other one for BANGBANG_TEMPLATE